.gitignore

```diff
@@ -68,3 +68,4 @@ ecr.repo
 .editorconfig
 helm_deploy/*/Chart.lock
 helm_deploy/*/charts/
+.trivyignore
```

Comment:
this shouldn't be ignored since the trivy file is required to be present during the trivy check